packages/web-components/src/components/va-telephone/va-telephone.tsx

```diff
@@ -87,9 +97,14 @@ export class VaTelephone {
     const numberType = vanity
       ? number.split(/[^\d]+/)
       : number.split(/[^\d\w]+/);
+
     return numberType
       .filter(n => n)
-      .map(chunk => (chunk === 'ext' ? 'extension' : chunk.split('').join(' ')))
+      .map(chunk => {
+        if (chunk === 'ext')
+          return 'extension';
+        return chunk === 'TTY' ? 'TTY' : chunk.split('').join(' ');
```

Comment:
If we keep adding more variants we may want to refactor all of these formatting functions because they're kind of getting unwieldy.

Reply:
Is it worth doing that refactor now?

Reply:
I mean, we already have it inside a named function:

https://github.com/department-of-veterans-affairs/component-library/blob/a17e132af6ba21a1727e50359d881739930d106a/packages/web-components/src/components/va-telephone/va-telephone.tsx#L88-L109

I could give that map function a name but I don't know if it would really do much. As we add more features, things like this get more unwieldy as well:

https://github.com/department-of-veterans-affairs/component-library/blob/a17e132af6ba21a1727e50359d881739930d106a/packages/web-components/src/components/va-telephone/va-telephone.tsx#L73-L80